tests/unit/fixtures/build_info/build_info

```diff
@@ -1,4 +1,4 @@
-Packaged by: jenkins@scalyr.com
-Latest commit: 7d4c4e2e94242ee25320a75c510d52967cfe50eb
-From branch: release
-Build time: 2020-05-06 17:59:21 UTC
+{
+    "build_info": "Packaged by: jenkins@scalyr.com\nLatest commit: 7d4c4e2e94242ee25320a75c510d52967cfe50eb\nFrom branch: release\nBuild time: 2020-05-06 17:59:21 UTC\n",
```

Comment:
Since this file now contains a JSON object, it may make sense to split this built info into multiple fields instead of using e raw string - e.g. packaged_by, revision_sha, branch_name, build_time or similar.

This may give us more flexibility and make it a bit easier to parse out the commit sha and similar (and we can still re-constructor single raw string from those values, if needed).